cost/aws/unused_rds/README.md

```diff
@@ -27,6 +29,8 @@ This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Auto
                   "cloudwatch:GetMetricStatistics",
                   "cloudwatch:GetMetricData",
                   "cloudwatch:ListMetrics",
+                  "rds:DescribeDBInstances",
+                  "rds:ListTagsForResource",
```

Comment:
```suggestion
                  "rds:ListTagsForResource"
```
remove ending `,` from JSON

Reply:
resolved as additional permission has been added to list